run_all.sh

```diff
@@ -1,23 +1,34 @@
-# Check the types in everything.
-pytype cabby
+REGION_NAME="Pittsburgh_small"
 
-# Geo
-bazel-bin/cabby/geo/geo_computation --orig_lat 40.749102 --orig_lon -73.984076 -dest_lat 40.748432 --dest_lon -73.982473
-bazel-bin/cabby/geo/map_processing/map_processor --region DC --min_s2_level 18 --directory "./cabby/geo/map_processing/poiTestData/"
-bazel-bin/cabby/geo/map_processing/map_processor --region Pittsburgh_small --min_s2_level 18 --directory "./cabby/geo/map_processing/poiTestData/"
-bazel-bin/cabby/geo/sample_poi --region DC --min_s2_level 18 --directory "./cabby/geo/map_processing/poiTestData/" --path "./cabby/geo/pathData/dc_geo_paths.gpkg" --n_samples 1
+echo "****************************************"
+echo "*                 Geo                  *"
+echo "****************************************"
+mkdir -p /tmp/poiTestData
 
-# RVS
-bazel build cabby/rvs:*
-bazel query cabby/rvs:* | xargs bazel test
-bazel-bin/cabby/rvs/generate_rvs --rvs_data_path "./cabby/geo/pathData/pittsburgh_geo_paths.gpkg" --save_instruction_path "./cabby/rvs/data/pittsburgh_instructions.json" 
+# Do we still need geo_computation?
+# bazel-bin/cabby/geo/geo_computation --orig_lat 40.749102 --orig_lon -73.984076 -dest_lat 40.748432 --dest_lon -73.982473
 
-# Wikidata
-bazel-bin/cabby/data/wikidata/extract_geofenced_wikidata_items --region Pittsburgh
+bazel-bin/cabby/geo/map_processing/map_processor --region $REGION_NAME --min_s2_level 18 --directory "/tmp/poiTestData/"
+bazel-bin/cabby/geo/sample_poi --region $REGION_NAME --min_s2_level 18 --directory "/tmp/poiTestData/" --path "/tmp/pathData/pittburgh_small_geo_paths.gpkg" --n_samples 1
```

Comment:
Region for sample_poi should b DC becuse Pittsburgh_small won' produce a path as it is too small.

Reply:
Fixed!